### PR TITLE
[ADD] Added bank* modules

### PR DESCRIPTION
**Pull request by JayVora-SerpentCS**

### CODE REVIEW

**Review by tafaRU — APPROVED** [review]
Travis errors are unrelated so :+1:

File: account_invoice_default_company_bank_repair/README.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+============================================
```
Comment (tafaRU) [inline account_invoice_default_company_bank_repair/README.rst]: Equal signs and title have to be same length, please remove one equal.

File: account_invoice_default_company_bank_repair/README.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Account Invoice Default Company Bank Repair
+============================================
```
Comment (tafaRU) [inline account_invoice_default_company_bank_repair/README.rst]: Same as above.

**Review by tafaRU — COMMENTED** [review]
Please rebase after have made the changes.
Thanks!

File: account_invoice_default_company_bank_repair/README.rst (hunk 2 above)
Comment (JayVora-SerpentCS) [reply account_invoice_default_company_bank_repair/README.rst]: Addressed, this was very strict :+1:

**Comment by tafaRU** [comment]
@JayVora-SerpentCS, please squash the commits to only one :wink:

**Comment by JayVora-SerpentCS** [comment]
Squashing causes issue while the base 8.0 is old and I had merged. Please let this proceed.

**Comment by tafaRU** [comment]
LGTM

**Review by NL66278 — APPROVED** [review]
LGTM :+1:

**Review by eLBati — APPROVED** [review]

**Pull request closed without merging**